locallibrary/catalog/templates/index.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base_generic.html" %} {% load i18n %} {% block content %}
```

Comment:
xuống dòng ở code tag cho clear e nhé

```suggestion
{% extends "base_generic.html" %}
{% load i18n %}

{% block content %}
```